cypress/support/pages/home.pageObject.js

```diff
@@ -7,10 +7,51 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
+  get authorName() {
+    return cy.getByDataCy('author-name-home-page');
+  }
+
+  get followBtn() {
+    return cy.getByDataCy('follow-btn');
+  }
+
+  get unfollowBtn() {
+    return cy.getByDataCy('unfollow-btn');
```

Comment:
Ensure that the `getByDataCy` command correctly returns the element with the data-cy attribute 'unfollow-btn'.